libs/blocks/src/lib/trade-types/index.tsx

```diff
@@ -22,10 +28,14 @@ export const TradeTypeBlock = <T extends CardVariantType>({
 }: TradeTypeBlockProps<T>) => {
   return (
     <Section className="bg-background-primary-container py-general-4xl">
-      <FluidContainer className="flex flex-col items-center justify-center gap-gap-xl">
-        <Heading.H2>{title}</Heading.H2>
+      <FluidContainer className="flex flex-col items-center justify-center">
+        <Heading.H2
+          className={qtJoin(description ? 'pb-gap-xl' : 'pb-gap-3xl')}
+        >
+          {title}
+        </Heading.H2>
         {description && (
-          <Text size="lg" className="text-typography-default">
+          <Text size="lg" className="pb-gap-3xl">
```

Comment:
please use `pb-general-[value]`

Reply:
Changed to `general` instead of using `gap`